src/api/app/assets/javascripts/webui/request_show_redesign/action_selector.js

```diff
@@ -0,0 +1,6 @@
+$(document).ready(function() {
+  $('.dropdown#request-actions').on('shown.bs.dropdown', function () {
```

Comment:
This works great :+1:.

Could we move this code to the view/component where it is needed? Otherwise, it's processed on every page, although nothing obviously happens since `$('.dropdown#request-actions')` doesn't match.